### PR TITLE
Explorer home mockup

### PR DESCRIPTION
**Pull request by befitsandpiper**
This is an intial pass at the Explorer Home FrontEnd. It does not include d3 work for the graph and bar chart, and is not perfectly to design spec. The APIs are also not hooked up, so the data is all fake.

There are likely at least 3 more PRs related to this page: 
- 1 for the d3 work, which will be the next main focus
- 1+ PRs for hooking up APIs and making the page functional
- a final PR to fine tune the designs and make sure everything's to spec.

here is the catch all ticket for this page: https://github.com/sourcecred/sourcecred/issues/2227

screencap:
image

### CODE REVIEW

**Review by topocount — APPROVED** [review]
Digging these changes, @befitsandpiper! I think this should be good to merge.

> There are likely at least 3 more PRs related to this page: 
> - 1 for the d3 work, which will be the next main focus
> - 1+ PRs for hooking up APIs and making the page functional
> - a final PR to fine tune the designs and make sure everything's to spec.

I recommend making an issues for each of these tasks and linking them back here, just so it's easier to track this process.

**Comment by befitsandpiper** [comment]
This PR covers the initial mockups for #2237, #2260, and #2231

Next PR will be based on this ticket: https://github.com/sourcecred/sourcecred/issues/2296

API connection ticket here: https://github.com/sourcecred/sourcecred/issues/2297

frontend design cleanup/polish ticket here: https://github.com/sourcecred/sourcecred/issues/2298

File: src/ui/components/ExplorerHome/ExplorerHome.js [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+    height: 0,
+    background: theme.palette.background,
+    borderTop: "30px solid transparent",
+    borderBottom: "30px solid transparent",
+    borderLeft: `30px solid ${theme.palette.background.paper}`,
+  },
+  circle: {
+    height: "128px",
+    width: "128px",
+    border: `1px solid ${theme.palette.text.primary}`,
```
Comment (KuraFire) [inline src/ui/components/ExplorerHome/ExplorerHome.js:52]: Let's set this to `5px` border width on the big circles, for now. :)

**Review by KuraFire — COMMENTED** [review]
**LGTM** 👍🏼

**Pull request merged**